README.md

```diff
@@ -21,6 +21,13 @@ xhr({
 ```
 
 ## `var req = xhr(options, callback)`
+Using this invocation, `options` should include a `uri` property for the request.
+
+## `var req = xhr(url, callback)`
```

Comment:
I merged from your repo and redacted some more. Check https://github.com/Raynos/xhr/pull/98/files